apps/export/views.py

```diff
@@ -26,14 +27,23 @@ def get(self, request, file_id=None, version=None):
         pass
 
 
-class ExportViewSet(viewsets.ReadOnlyModelViewSet):
+class ExportViewSet(mixins.DestroyModelMixin,
+                    mixins.ListModelMixin,
+                    mixins.RetrieveModelMixin,
+                    viewsets.GenericViewSet):
     serializer_class = ExportSerializer
     permission_classes = [permissions.IsAuthenticated]
     filterset_class = ExportFilterSet
 
     def get_queryset(self):
         return Export.get_for(self.request.user)
```

Comment:
This should filter out the soft-deleted Exports. Add filter is_deleted=False in get_for function's queryest.

Reply:
Also, add test for this too.